server.js

```diff
@@ -8,7 +8,7 @@ const HOST = '0.0.0.0';
 
 const LT_VERSION = '2018-05-01';
 const LT_URL = process.env.lt_url || 'https://gateway.watsonplatform.net/language-translator/api';
-const PORT = process.env.PORT || 3000;
+//const PORT = process.env.PORT || 3000;
```

Comment:
@jjasghar any way we can run the node app via travis and abandon the push if the app does not run?

I think @stevemar added this to assign a default port. We can either leave this in or change the earlier line to 

```suggestion
const PORT = process.env.PORT || 8080;
const HOST = '0.0.0.0';


const LT_VERSION = '2018-05-01';
const LT_URL = process.env.lt_url || 'https://gateway.watsonplatform.net/language-translator/api';
//const PORT = process.env.PORT || 3000;
```

If we keep @stevemar port 3000, let's change docker file to 
```
EXPOSE 3000
CMD [ "node", "server.js" ]
```

Reply:
@lidderupk thank you for accepting the PR. if you expose a different port in a Docker file you are going to break my super cool instructions :-) make sure it is reflected in the readme.md

Reply:
@blumareks, I didn't accept or merge. I think you did? I just added a comment to your PR to start a discussion 😨 🙏🏼

`blumareks merged commit 224a078 into master 12 minutes ago`

My suggestion would be to remove the line instead of commenting it out. Commenting out will confuse the audience.